client/packages/system/src/Patient/api/api.ts

```diff
@@ -30,22 +30,18 @@ export const getPatientQueries = (sdk: Sdk, storeId: string) => ({
     }> => {
       const key =
         sortBy?.key === 'code'
-          ? NameSortFieldInput.Code
-          : NameSortFieldInput.Name;
+          ? PatientSortFieldInput.Code
+          : PatientSortFieldInput.Name;
 
       const result = await sdk.patients({
         first,
         offset,
         key,
         desc: !!sortBy?.isDesc,
         storeId,
-        filter: {
-          isStore: false,
-          isSystemName: false,
-        },
```

Comment:
This should be being filtered on the back-end. It was earlier, but a recent PR (#102 ) regressed this and it's currently letting stores through. Issue to fix here: #103